sdk/interchain/README.md

```diff
@@ -0,0 +1,61 @@
+# Signature-aggregator SDK 
+
+## Usage example
+
+```
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/sdk/interchain"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func main() {
+	network := models.NewFujiNetwork()
+	peerNetwork, err := interchain.CreateAppRequestNetwork(network, logging.Error)
+	if err != nil {
+		panic(err)
+	}
+
+	aggregator, err := interchain.NewSignatureAggregator(
+		peerNetwork,
+		logging.NewLogger(
+			"aggregator_test",
+			logging.NewWrappedCore(
+				logging.Error,
+				os.Stdout,
+				zapcore.NewConsoleEncoder(
+					zap.NewProductionEncoderConfig(),
+				),
+			),
+		),
+		ids.Empty,
+		interchain.DefaultQuorumPercentage,
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	msg := "0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000002cc0000000000057fc93d85c6d62c5b2ac0b519c87010ea5294012d1e407030d6acd0021cac10d5000002a200000000000100000014253b2784c75e510dd0ff1da844684a1ac0aa5fcf000002800000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000049d0000000000000000000000009297e09b81db7ae921fb69dc745dc0d37ec4ab6ddd30039ae180ccb3b132f5c67f299b83749f1f0a12f7d69878c6b696598fd0c50000000000000000000000006839ebb9c7b7df12a2147790f6aaa19ac1bf76eb000000000000000000000000000000000000000000000000000000000001117000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000120000000000000000000000000000000000000000000000000000000000000018000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000600000000000000000000000014c3ae49cb46c965e6c9aca4a121c8f8481d44b900000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000040000000000000000000000000f7571ae49399c8a1b4c8c8eb8cc328bc5f817590000000000000000000000000000000000000000000000000016345785d8a00000000000000000000000000000000000000000000"
```

Comment:
can you also add a var for the justification. and if possible, explain why we need a msg + a justification

Reply:
done